packages/vscode-integration/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@eclipse-glsp/vscode-integration",
   "displayName": "GLSP VSCode Integration",
-  "version": "2.1.0",
+  "version": "2.2.0-next",
```

Comment:
Could you please shortly elaborate why we need to set this to `2.2.0-next`? I see that a lot of dependencies change in this PR. How should adopters use this with all the `2.1.1` release versions out there? Does this make a resolutions block necessary?

Reply:
The master does not reflect the 2.2.1 hotfix branch. It reflects the current next iteration (2.2.0 in this case).
Typcially we directly change back the dependencies to the "next" versions after a release (e.g. https://github.com/eclipse-glsp/glsp-client/commit/b90a74030c271dd26d66dc1f88987661aeab24c3) . In this case, we overlooked it, so I had to this as part of this PR.

Once merged, this PR will be backported on top of the 2.1.0 release and the 2.2.1 hotfix version will be released.

Reply:
I understand, thank you for the explanation!